cmd/up/up.go

```diff
@@ -274,7 +274,15 @@ func (up *upContext) start() error {
 	stop := make(chan os.Signal, 1)
 	signal.Notify(stop, os.Interrupt)
 
-	analytics.TrackUp(true, up.Dev.Name, up.getInteractive(), len(up.Dev.Services) == 0, up.Dev.Divert != nil)
+	cwd, err := os.Getwd()
+	if err != nil {
+		oktetoLog.Infof("failed to get the current working directory in up: %v", err)
+	}
+	repo, err := model.GetRepositoryURL(cwd)
+	if err != nil {
+		oktetoLog.Info("could not get repository from current dir")
+	}
+	analytics.TrackUp(true, up.Dev.Name, repo, up.getInteractive(), len(up.Dev.Services) == 0, up.Dev.Divert != nil)
```

Comment:
@rlamana let's send if the repo belongs to the okteto organization or not

Reply:
Yeah, I thought we already agreed on that. How are we going to name that property?

Reply:
`isDemoApp`?

Reply:
`useDemoApp`?
`useOktetoApp`?
`useOktetoSampleApp`?

All good to me, as long as it describes clearly what it is for those who will use mixpanel in the future, and they can intuitively interpret what the property means...

Reply:
+1 to `isOktetoApp`. I think it's clear that you are using an okteto app or we can even use `isOktetoRepository` so we know exactly what this property refers to. WDYT @rlamana @pchico83

Reply:
SGTM

Reply:
👍